packages/lab/src/pill/internal/PillBase.css

```diff
@@ -19,60 +19,65 @@
   --pill-background: var(--uitkPill-background, var(--uitk-taggable-background));
   --pill-background-active: var(--uitkPill-background-active, var(--uitk-taggable-background-active));
   --pill-background-disabled: var(--uitkPill-background-disabled, var(--uitk-taggable-background-disabled));
-  --pill-background-hover: var(--uitkPill-background-hover, var(--uitk-taggable-background));
+  --pill-background-hover: var(--uitkPill-background-hover, var(--uitk-taggable-background-hover));
+
   --pill-checkbox-size: var(--uitkPill-checkbox-size, calc(var(--uitk-size-pill) - 2px));
+
   --pill-icon-color: var(--uitkPill-icon-color, var(--uitk-taggable-foreground));
   --pill-icon-color-active: var(--uitkPill-icon-color-active, var(--uitk-taggable-foreground-active));
   --pill-icon-color-hover: var(--uitkPill-icon-color-hover, var(--uitk-taggable-foreground-hover));
+
   --pill-text-color: var(--uitkPill-text-color, var(--uitk-taggable-foreground));
   --pill-text-color-active: var(--uitkPill-text-color-active, var(--uitk-taggable-foreground-active));
   --pill-text-color-hover: var(--uitkPill-text-color-hover, var(--uitk-taggable-foreground-hover));
 }
 
-/* Button Component API - reset to default state so they don't interfere with Pill styling */
+/* Button Component API - use Pill variables so the default doesn't interfere with Pill styling */
 .uitkPill {
-  --uitkButton-background-active: var(--pill-background);
-  --uitkButton-textColor-active: var(--pill-text-color);
-  --uitkButton-background-hover: var(--pill-background);
-  --uitkButton-textColor-hover: var(--pill-text-color);
-  --uitkButton-font-weight: var(--uitkPill-button-font-weight, var(--uitk-typography-weight-regular));
+  --uitkButton-font-weight: var(--uitkPill-button-fontWeight, var(--uitk-typography-weight-regular));
   --uitkButton-padding: var(--uitkPill-button-padding, 0);
-  --uitkButton-textTransform: var(--uitkPill-button-text-transform, unset);
+  --uitkButton-textTransform: var(--uitkPill-button-textTransform, unset);
   --uitkButton-cursor: normal;
 }
 
+.uitkPill:hover:not(.uitkPill-selectable):not(.uitkPill-deletable) {
```

Comment:
Something to think about - `:not` increases specificity making consumer overrides harder (including a few other places down). Whether we can somehow restructure the CSS to make basic overrides less specific. This maybe an exercise beyond just CSS but also involve how classes are applied in React code.